pontoon/teams/static/js/request.js

```diff
@@ -0,0 +1,182 @@
+/* Public functions used across different files */
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    requestItem: {
+      /*
+       * Toggle available projects/teams and request div
+       *
+       * show Show enabled projects/teams?
+       */
+      toggleItem: function (show, type) {
+        // Toggle
+        $('.controls .request-toggle')
+          .toggleClass('back', !show)
+          .find('span')
+            .toggleClass('fa-chevron-right', show)
+            .toggleClass('fa-chevron-left', !show);
+
+        if (type === 'projects') {
+          var localeProjects = $('#server').data('locale-projects');
+          // Hide all projects
+          $('.projects')
+            .toggleClass('request', !show)
+            .find('tbody tr')
+              .toggleClass('limited', !show)
+              .toggle(!show);
+
+          // Show requested projects
+          $(localeProjects).each(function() {
+            $('.projects')
+                .find('td[data-slug="' + this + '"]')
+              .parent()
+                .toggleClass('limited', show)
+                .toggle(show);
+          });
+          Pontoon.requestItem.toggleButton(!show, 'projects');
+        } else if (type === 'team') {
```

Comment:
There are a few please in this file, where we should change this style:

```js
if () {
  // something
} else {
  // something else
}
```
To this:
```js
if () {
  // something
}
else {
  // something else
}
```